redisgraph/crates/wasccgraph-actor/Cargo.toml

```diff
@@ -1,18 +1,20 @@
 [package]
-name = "wasccgraph-actor"
-version = "0.0.3"
+name = "redisgraph-actor"
```

Comment:
I'm confused here. If we have the actor interface in `actor-interfaces/graphdb` , then why do we have another graph actor API that's specific to redis?

We should have a vendor-neutral graphdb actor interface in the actor-interfaces repository, and then we should have the stuff that _only_ applies to Redis in the redisgraph repository (the capability provider implementation).